File: Practice/11/C++/degree/degree/degree.cpp
#include <iostream>
using namespace std;

int main()
{
	setlocale(LC_ALL, "Russian");

	double defnum, num;
	int deg, cnt;
	cout << "Введите число и степень, в которую его нужно возвести\n";
	cin >> defnum;
	cin >> deg;
	num = defnum;
	if (deg == 0)
		cout << 1;
	else if (deg < 0)
	{
		deg = -deg;
		for (cnt = 1; cnt < deg; cnt = cnt + 1)
			num = num * defnum;
		cout << 1 / num;
	}
	else
	{
		for (cnt = 1; cnt < deg; cnt = cnt + 1)
			num = num * defnum;
		cout << num;
	}
}

File: Practice/11/Python/degree.py
print('Введите число и степень, в которую его надо возвести')
defnum = float(input())
deg = int(input())
num = defnum
if deg == 0:
    print(1)
elif deg < 0:
	deg = -deg
	for cnt in range(1, deg):
		num = num * defnum
	print(1 / num)
else:
	for cnt in range(1, deg):
		num = num * defnum
	print(num)